CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # CHANGELOG
 
 
+## vN.N.N
+* issue #1238: cms will crash after session/raven disconnect and users accept new disconnect Alert
```

Comment:
@hunyoboy remember we dont link to tickets that are not part of this repo

Reply:
@efjacobson removed the link and updated changed log.